packages/react/src/components/Typography/Heading/Heading.tsx

```diff
@@ -31,7 +31,7 @@ export type HeadingProps = {
  */
 export const Heading = forwardRef<HTMLHeadingElement, HeadingProps>(
   (
-    { size = 'xl', level = 1, spacing = false, className, asChild, ...rest },
+    { size = 'xl', level = 2, spacing = false, className, asChild, ...rest },
     ref,
   ) => {
     const Component = asChild ? Slot : (`h${level ?? 1}` as ElementType);
```

Comment:
Why do we even have this fallback check here?

Reply:
Not sure, I can remove it